### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by msonowal**

### CODE REVIEW

File: src/ShopifyApp/resources/config/shopify-app.php [hunk 1]
```diff
@@ -206,6 +206,19 @@
...
 
     'api_rate_limit_cycle_buffer' => env('SHOPIFY_API_RATE_LIMIT_CYCLE_BUFFER', null),
+    
+    /*
+    |--------------------------------------------------------------------------
+    | Shopify Jobs Namespace
+    |--------------------------------------------------------------------------
+    |
+    | This option option allows you to change out the default job namespace
+    | which is \App\Jobs. This option is mainly used if any custom configuration
```
Comment (bilfeldt) [inline src/ShopifyApp/resources/config/shopify-app.php]: Fails StyleCi. Please remove trailing space.

File: src/ShopifyApp/resources/config/shopify-app.php [hunk 2]
```diff
@@ -206,6 +206,19 @@
...
+    | Shopify Jobs Namespace
+    |--------------------------------------------------------------------------
+    |
+    | This option option allows you to change out the default job namespace
+    | which is \App\Jobs. This option is mainly used if any custom configuration 
+    | is done in autoload and does not need to be changed unless required.
+    |
+    */
+
+    'job_namespace' => env('SHOPIFY_JOB_NAMESPACE', '\OhMyBrew\ShopifyApp\Jobs\\'),
```
Comment (bilfeldt) [inline src/ShopifyApp/resources/config/shopify-app.php]: You are using `'\OhMyBrew\ShopifyApp\Jobs\\'` but the jobs will be placed in `\App\Jobs\`. Changing to
```php
env('SHOPIFY_JOB_NAMESPACE', '\\App\\Jobs\\')
```
should fix the test `WebhookControllerTest`

**Comment by ohmybrew** [comment]
All tests are failing for some reason, any insights to why?

**Comment by ohmybrew** [comment]
I will merge and adjust the namespace manually, thanks!

**Pull request merged**